### ISSUE TITLE
Delete TOP_EU_US_Ads_Trackers_HOST

### ISSUE THREAD

**Issue by dmginc** [Kees1958/W3C_annual_most_used_survey_blocklist#55]
Was this intentional? Is it being replaced by something? I didn't see any notes in the commit.

**Comment by Dean151** [comment]
Was deleted by https://github.com/Kees1958/W3C_annual_most_used_survey_blocklist/commit/3427e2b371254f75f29fd70a8166476b3199ea20

I'd love this to be not intentional and have it to come back updated, because the HOST file is useful for pihole blocking

**Comment by TwisX** [comment]
Also using this for PiHole. Hoping this was a mistake.

**Comment by Blfrg** [comment]
A [previous commit](https://github.com/Kees1958/W3C_annual_most_used_survey_blocklist/commit/c5ded06c6e96dbd63af8a92562dc63d9cdbaa39f) suggests this was intentional.

---

Confirmed the remaining _ABP_ list is not sufficient for Pi-hole:
```js
[i] Target: https://raw.githubusercontent.com/Kees1958/W3C_annual_most_used_survey_blocklist/master/TOP_EU_US_Ads_Trackers_ABP
  [✓] Status: Retrieval successful
  [i] Analyzed 3527 domains, 3526 domains invalid!
```

I had already deleted my _HOST_ version, if you need to add it back, [this](https://raw.githubusercontent.com/Kees1958/W3C_annual_most_used_survey_blocklist/b3a54bf1ce5a48d62afada529bd900c2f2e587ba/TOP_EU_US_Ads_Trackers_HOST) was the final one

**Comment by travisboss** [comment]
I am guessing the author moved to adguard home instead of pihole?  I caught this on my compiled list for AGH and moved to the ABP syntax.  All I can say is thanks!

**Comment by Kees1958** [comment]
Sorry guys, the host file was something initiated by Beerisgood, since he did not want to contribute this anymore, I removed this host file since I am not using it myself. As said in 2019 I used a source which I rejected, but I accidentally included in a mass update. Because I threw away the rejected source and previous version, I could not back track the changes. 

Manually checking both the ABP and HOST was to much of an effort. There for I decided to drop the HOST file support.

Apologize for the inconvenience

Regards Kees
